res/tmpl/index.html

```diff
@@ -35,19 +35,15 @@
             <li><a href="#" id="outbox_tab">Outbox</a></li>
             <li><a href="#" id="sent_tab">Sent</a></li>
             <li><a href="#" id="archive_tab">Archive</a></li>
-            <li class="dropdown">
+<!--          Flattened menu as the additional dropdown was annoying 
```

Comment:
The comment is not needed when merged, I think this should be removed.

Reply:
Fair enough